README.md

```diff
@@ -8,6 +8,97 @@ software below.
 
 ----------------------------------------------------------------------
 
+Installation Instructions
```

Comment:
Make use of markdown syntax to highlight headings and code, create lists (numbered or bullets), use links if necessary, etc.